com.kassindornelles.lolforlinuxinstaller.yaml

```diff
@@ -35,6 +35,8 @@ modules:
 
       build-commands:
         #install the launcher to the correctly location
+        - ls -a
+        - ls ./src/
         - install -D ./src/com.kassindornelles.lolforlinuxinstaller.py /app/bin/com.kassindornelles.lolforlinuxinstaller
```

Comment:
file names are case sensitive.

![image](https://github.com/flathub/flathub/assets/114441/9636253b-f174-426c-b2e6-eefdb1e9047c)

This also probably apply to the line below too.

Reply:
it's fixed in the git HEAD, just waiting for the release